apps/spruce/src/analytics/projectHealth/useProjectHealthAnalytics.ts

```diff
@@ -3,7 +3,11 @@ import { AnalyticsIdentifier } from "analytics/types";
 import { ProjectHealthView } from "gql/generated/types";
 
 // The comments below are used to indicate which pageType the action is relevant to (e.g. "Commit chart")
-type pageType = "Commit chart" | "Task history" | "Variant history";
+type pageType =
+  | "Commit chart"
+  | "Task history"
+  | "Variant history"
+  | "Waterfall";
```

Comment:
How do you feel about just creating an entirely new analytics object/hook for this? 
It's a new page and we are moving away from the project health verbiage it could be nice to start over with these analytics attributes.

Reply:
That does make sense, I thought it could be nice to avoid redundancy but I think a separate object will ultimately be easier to manage. Will do!